tests/providers/amazon/aws/triggers/test_redshift_cluster.py

```diff
@@ -500,3 +501,42 @@ async def test_redshift_resume_cluster_trigger_run_attempts_failed(
         assert response == TriggerEvent(
             {"status": "failure", "message": f"Resume Cluster Failed: {error_failed}"}
         )
+
+
+class TestRedshiftDeleteClusterTrigger:
+    class TestRedshiftDeleteClusterTrigger:
```

Comment:
Can you add test cases for multiple attempts, max_attempts reached, and a failure case?
https://github.com/apache/airflow/blob/main/tests/providers/amazon/aws/triggers/test_redshift_cluster.py#L120

Reply:
Added